### PR TITLE
[Feature] Support ShuffleNetV2

### PR DESCRIPTION
**Pull request by Junjun2016**
## Motivation

Support more real-time semantic segmentation models.
In this PR, we add ShuffleNetV2.

## Modification

- [x] Add ShuffleNetV2 backbone and unitests
- [ ] Add configs for ShuffleNetV2
- [ ]  Add benchmarks for ShuffleNetV2

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos? 
No

### CODE REVIEW

File: tests/test_models/test_backbones/test_shufflenet_v2.py [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+        assert param.requires_grad is False
+    for i in range(0, frozen_stages):
+        layer = model.layers[i]
+        for mod in layer.modules():
+            if isinstance(mod, _BatchNorm):
+                assert mod.training is False
+        for param in layer.parameters():
+            assert param.requires_grad is False
+
+    # Test ShuffleNetV2 with first stage frozen, frozen_stages=4
```
Comment (Junjun2016) [inline tests/test_models/test_backbones/test_shufflenet_v2.py:109]:
```suggestion
    # Test ShuffleNetV2 with all stages frozen, frozen_stages=4
```

File: mmseg/models/backbones/shufflenet_v2.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+                conv_cfg=conv_cfg,
+                norm_cfg=norm_cfg,
+                act_cfg=act_cfg))
+
+    def _make_layer(self, out_channels, num_blocks, stride, dilation):
+        """Stack blocks to make a layer.
+
+        Args:
+            out_channels (int): out_channels of the block.
+            num_blocks (int): number of blocks.
```
Comment (Junjun2016) [inline mmseg/models/backbones/shufflenet_v2.py:266]: Missing docstring of all `args`.
